src/main/resources/templates/team.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org" xmlns:sec="http://www.thymeleaf.org/extras/spring-security">
+
+    <head th:replace="c_head :: head"></head>
+
+    <body>
+
+        <header th:replace="c_header :: header"></header>
+
+        <!--<header th:replace="c_nav_m :: nav_m"></header>-->
+
+        <!--Basic group information-->
+        <div class="shades">
+            <span th:text="'Name:' + ${team}"></span>
```

Comment:
These spans are REALLY hard for users to read/parse because they're all on one line. At least use divs, or consider styling harder!